src/engine.rs

```diff
@@ -382,15 +387,15 @@ impl Engine {
         value: Wei,
         input: Vec<u8>,
     ) -> EngineResult<Vec<u8>> {
-        let mut executor = self.make_executor();
+        let mut executor = self.make_executor(u64::MAX);
```

Comment:
Do view calls need a gas limit?

Reply:
Yes, for compatibility with mutative calls, a gas limit is passed in: https://eth.wiki/json-rpc/API#eth_call

Reply:
Gas limit added to `Engine::view` in https://github.com/aurora-is-near/aurora-engine/pull/123/commits/b8879c8dd53cb2c8b542180432d55003158e69ab but it's not really used yet because the only way `view` is called is from `view_with_args` which does not have a gas limit (since it is Near-native). However, we will add an `eth_call` contract method in the future which will pass a user-input gas limit to `Engine::view`.